core/tests/webdriverio_desktop/topicAndStoryEditor.js

```diff
@@ -382,3 +385,119 @@ describe('Chapter editor functionality', function() {
     }
   });
 });
+
+describe('Serial Chapter Launch functionality', function() {
+  var topicsAndSkillsDashboardPage = null;
+  var topicEditorPage = null;
+  var storyEditorPage = null;
+  var releaseCoordinatorPage = null;
+  var adminPage = null;
+  var storyName = 'Story 0';
+  var explorationEditorPage = null;
+  var dummyExplorationIds = [];
+  var dummyExplorationInfo = [
+    'Dummy exploration', 'Algorithms', 'Learn more about oppia', 'English'];
+  var topicName = 'Topic 0';
+  var topicUrlFragment = 'topic-zero';
+  var userEmail = 'creator@serialChapterTest.com';
+
+  var createDummyExplorations = async function(numExplorations) {
```

Comment:
(optional) If you see this method in other places, can we move it to utils file?